malloc/malloc.c

```diff
@@ -26,7 +29,13 @@ char *string_dup(char *src)
 */
 void *mem_copy(void *dest, const void *src, int n)
 {
-
+    char *charSrc = (char *) src;
```

Comment:
There is ofc, also a cast here which I'm more familiar with for casting those void pointers to char pointers.